app/helpers/image_service_helper.rb

```diff
@@ -82,7 +104,7 @@ def member_image_tag(parent_id:, member:, size_key: nil, lazy: false, use_image_
 
   private
 
-  def create_riiif_url(path)
+  def create_iiif_url(path)
     if CHF::Env.lookup(:public_riiif_url)
       url = Addressable::URI.parse(CHF::Env.lookup(:public_riiif_url))
```

Comment:
Maybe change these ENVs to public_iiif_url and internal_iiif_url too, with a switch on iiif_type or whatever?